Add-IN/ACDC4Robot/core/urdf.py

```diff
@@ -539,7 +548,7 @@ def get_link_inertial_origin(link: Link) -> list[float]:
         unit: m, radian
     """
     # the link is the first link so called base-link
-    if link.get_parent_joint() is None:
+    if not joint_has_geometry(link.get_parent_joint()):
```

Comment:
This will be false if there is no parent joint, or if it's an as-built rigid joint.